CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 1.3.0 - 2017-10-18
```

Comment:
It's 16th today, why 18? :)

Reply:
Timetraveling you know they have this in the future ...